ocs_ci/helpers/dr_helpers.py

```diff
@@ -73,3 +73,25 @@ def relocate(preferred_cluster, drpc_name, namespace):
     drpc_obj.wait_for_phase(constants.STATUS_RELOCATED)
 
     config.switch_ctx(prev_index)
+
+
+def check_mirroring_status(pattern_key="health", exp_pattern_value="OK"):
+    """
+    Validates current mirroring status matches with expected values
+
+    """
+    cbp_obj = ocp.OCP(
+        kind=constants.CEPHBLOCKPOOL,
+        resource_name=constants.DEFAULT_CEPHBLOCKPOOL,
+        namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+    )
+    mirroring_status = cbp_obj.get().get("status").get("mirroringStatus").get("summary")
+    pattern_value = mirroring_status.get(pattern_key)
+    logger.info(f"Current status of {pattern_key} is {pattern_value}")
+    if pattern_key == "states":
+        pattern_value = pattern_value.get("replaying")
```

Comment:
I see the default pattern_key as "health"
if "health" == "states"
so this will never execute "pattern value = pattern_value.get("replaying")

Please keep the default pattern_key as "None" and pass the pattern_keys from the testcase. Line 92 will never be executed when pattern_key is "health"